apig/generate.go

```diff
@@ -487,15 +487,19 @@ func detectImportDir(targetPath string) (string, error) {
 
 	importDir := formatImportDir(importPaths)
 
-	if len(importDir) > 1 {
+	if len(importDir) > 2 {
 		return "", errors.New("Conflict import path. Please check 'main.go'.")
 	}
 
-	if len(importDir) == 0 {
+	if len(importDir) < 2 {
 		return "", errors.New("Can't refer import path. Please check 'main.go'.")
 	}
 
-	return importDir[0], nil
+	if importDir[0] != "." {
```

Comment:
Since `TestFormatImportDir(t *testing.T)` was also considered `.`, I thought that `.` is also necessary to count.
If `apig` don't need `.`, I remove `.` in `parseImport`.